### DISCUSSION TITLE
Do we have plan to support Siglip model for new llava-next-video architecture?

### DISCUSSION THREAD

**Discussion by LetheRiver0** [sgl-project/sglang#777] [General]
Hi, I notice llava-next has published new version of llava-next-video model with llava-qwen and siglip vision tower, I wonder do have plan to support siglip in sglang? Thanks~

**Comment by Ying1123** [comment]
cc @ZhangYuanhan-AI @Luodian

**Comment by Luodian** [comment]
Yes, we do have this implementation and will PR it later.👨‍💻

**Reply by merrymercy** [reply]
https://github.com/sgl-project/sglang/pull/1004

**Comment by LetheRiver0** [comment]
That will be an amazing work! Looking forward to it~

**Comment by merrymercy** [comment]
done by #1004
